docs/userguide/icu/editing.md

```diff
@@ -159,6 +159,7 @@ To activate the version of Ruby:
 ```bash
 rbenv init  # OR:  eval "$(rbenv init -)"
 rbenv shell <version-num>
+rbenv versions  # verify the specified version is in use
```

Comment:
Whole-file comment: From the navbar grouping (which makes sense to me), it looks like this should not be at icu/editing.md but rather at dev/editing.md